packages/venia-concept/src/actions/cart/asyncActions.js

```diff
@@ -213,6 +213,7 @@ export const removeItemFromCart = payload => {
         dispatch(actions.removeItem.request(payload));
 
         const { cart, user } = getState();
+        let lastItemCount = false;
```

Comment:
This variable isn't a count, so let's give it a more descriptive name; perhaps `isLastItem`?

Reply:
Resolved by a14eec2